### PR TITLE
remove qualimap/rnaseq

### PR DESCRIPTION
**Pull request by rannick**
replace qualimap/rnaseq with mosdepth as mosdepth requires less cpu and memory

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by fevac** [comment]
possibly best to just remove qualimap and not replace it with anything (picard already provides the gene body coverage information). Waiting to make a decision

File: nextflow_schema.json [hunk 1]
```diff
@@ -363,6 +363,14 @@
                     "pattern": "^\\S+\\.interval_list?$",
                     "description": "Path to ribosomal interval list.",
                     "fa_icon": "far fa-file-code"
+                },
+                "rrna_intervals_bed": {
```
Comment (fevac) [inline nextflow_schema.json:367]: remove?

File: subworkflows/local/qc_workflow.nf [hunk 2]
```diff
@@ -17,14 +16,11 @@ workflow QC_WORKFLOW {
         ch_fasta
         ch_fai
         ch_rrna_interval
+        ch_rrna_intervals_bed
```
Comment (fevac) [inline subworkflows/local/qc_workflow.nf:19]: remove

File: workflows/rnafusion.nf [hunk 3]
```diff
@@ -27,6 +27,7 @@ ch_starindex_ref = params.starfusion_build ? Channel.fromPath(params.starindex_r
 ch_starindex_ensembl_ref = Channel.fromPath(params.starindex_ref).map { it -> [[id:it.Name], it] }.collect()
 ch_refflat = params.starfusion_build ? Channel.fromPath(params.refflat).map { it -> [[id:it.Name], it] }.collect() : Channel.fromPath("${params.ensembl_ref}/ref_annot.gtf.refflat").map { it -> [[id:it.Name], it] }.collect()
 ch_rrna_interval = params.starfusion_build ?  Channel.fromPath(params.rrna_intervals).map { it -> [[id:it.Name], it] }.collect() : Channel.fromPath("${params.ensembl_ref}/ref_annot.interval_list").map { it -> [[id:it.Name], it] }.collect()
+ch_rrna_interval_bed = params.starfusion_build ?  Channel.fromPath(params.rrna_intervals_bed).map { it -> [[id:it.Name], it] }.collect() : Channel.fromPath("${params.ensembl_ref}/Homo_sapiens.GRCh38.102.bed").map { it -> [[id:it.Name], it] }.collect()
```
Comment (fevac) [inline workflows/rnafusion.nf:30]: remove

File: workflows/rnafusion.nf [hunk 4]
```diff
@@ -263,7 +264,8 @@ workflow RNAFUSION {
         ch_refflat,
         ch_fasta,
         ch_fai,
-        ch_rrna_interval
+        ch_rrna_interval,
+        ch_rrna_interval_bed
```
Comment (fevac) [inline workflows/rnafusion.nf:268]: remove?

**Review by fevac — CHANGES_REQUESTED** [review]
Nice! 🌟 I think it would be good to rename the PR to avoid future confusion. Then just minor comments about removing rrna bed since it doesn't seem to be used anymore

**Comment by rannick** [comment]
Changes addressed

**Review by fevac — APPROVED** [review]
🚀

**Pull request merged**